test/integration/an_existing_user_can_check_out_to_place_an_order_test.rb

```diff
@@ -30,7 +30,7 @@ class AnExistingUserCanCheckOutToPlaceAnOrderTest < ActionDispatch::IntegrationT
     assert_equal '/orders', current_path
 
     within('table') do
-      assert page.has_content?('completed')
+      assert page.has_content?('ordered')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.